app/jobs/ready_for_cataloging_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Deliver the Ready For Cataloging email
+class ReadyForCatalogingJob < ApplicationJob
+  queue_as :ready_for_cataloging
```

Comment:
I'm curious how much we care what queue this runs in.

Reply:
Can likely ditch this whole thing.